src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMigrationFilter.kt

```diff
@@ -11,4 +12,10 @@ class PrisonPersonMigrationFilter(
     example = "A1234BC",
   )
   val prisonerNumber: String?,
+
+  @Schema(
+    description = "The type of migration for this person, e.g. which data is being migrated",
+    example = "PHYSICAL_ATTRIBUTES",
+  )
+  val migrationType: PrisonPersonMigrationMappingRequest.MigrationType = PrisonPersonMigrationMappingRequest.MigrationType.PHYSICAL_ATTRIBUTES,
```

Comment:
Pass the migration type into the start migration request.

Default to physical attributes so the UI continues to work (but will be changed later).

Not 100% sure about reusing the mapping migration type here - but duplicating the exact same values seems wrong too 🤔

Reply:
Interesting - not really had different migration types before